mix.exs

```diff
@@ -19,13 +19,13 @@ defmodule FakeArtist.Mixfile do
   def application do
     [
       mod: {FakeArtist.Application, []},
-      extra_applications: [:logger, :runtime_tools]
+      extra_applications: [:logger, :runtime_tools, :gen_state_machine]
```

Comment:
you shouldn't have to list `gen_state_machine` here, even if their README told you to. Please try removing it to see if it still works